src/components/dropDownStops.tsx

```diff
@@ -0,0 +1,83 @@
+import { Link } from '@react-navigation/native'
+import { StackHeaderProps } from '@react-navigation/stack'
+import { useState } from 'react'
+import { Linking, StyleSheet, View } from 'react-native'
+import { Appbar, Button, Text } from 'react-native-paper'
+import { getEqualHitSlop } from '../helpers/hitSlopHelper'
+import DropDown from 'react-native-paper-dropdown'
+
+export default function StopsDropDown() {
+  const [dropDownVisible, setDropDownVisible] = useState(true)
+  const [stop, setStop] = useState('')
+  const listOfStops = [
+    {
+      label: 'Oldham Mumps',
+      value: 'oldham-mumps-tram',
+    },
+    {
+      label: 'Peel Hall',
+      value: 'peel-hall-tram',
+    },
+    {
+      label: 'Freehold',
+      value: 'freehold-tram',
+    },
+    {
+      label: 'Velopark',
+      value: 'velopark-tram',
+    },
+    {
+      label: 'Anchorage',
+      value: 'anchorage-tram',
+    },
+    {
+      label: "Besses o' th' Barn",
+      value: 'besses-o-th-barn-tram',
+    },
+    {
+      label: 'Wythenshawe Town Centre',
+      value: 'wythenshawe-town-centre-tram',
+    },
+  ]
```

Comment:
`listOfStops` doesn't need to be in the component, move it to before 9 to declutter the component.

Also, is there a smarter way to deal with mapping the label to the value? What if I now ask you to add Cornbrook to the list?